config/templates/config-docker.conf

```diff
@@ -1,13 +1,17 @@
 # DO NOT EDIT THIS FILE
 #
 # This is a Docker launcher file. To set up the configuration, use command line arguments to compile.sh
-# or create a config file named "config-docker-guest.conf" based on config-example.conf
+# or use pass a config file as a parameter ./compile docker [example] BUILD_KERNEL="yes" ...
 
 [[ ! -c /dev/loop-control ]] && display_alert "/dev/loop-control does not exist, image building may not work" "" "wrn"
 
 # remove "docker" from the command line since "docker-guest" will be passed instead
 shift
 
+# second argument can be a build parameter or a config file
```

Comment:
Setting environment variables after command is **not** allowed in **Bourne shell** but in **GNU make**.

`# DOWNLOAD_MIRROR=china ./compile.sh` :heavy_check_mark:
`# ./compile.sh DOWNLOAD_MIRROR=china` :x:

Therefore, the second argument **must** be a config file.